node/src/accountant/mod.rs

```diff
@@ -4851,13 +4905,70 @@ mod tests {
 
         System::current().stop();
         system.run();
-        let fingerprints_rowids_params = fingerprints_rowids_params_arc.lock().unwrap();
-        assert_eq!(*fingerprints_rowids_params, vec![vec![expected_hash]]);
-        let mark_pending_payables_rowids_params =
-            mark_pending_payables_rowids_params_arc.lock().unwrap();
+        let inserted_new_records_params = inserted_new_records_params_arc.lock().unwrap();
         assert_eq!(
-            *mark_pending_payables_rowids_params,
-            vec![vec![(expected_wallet, expected_rowid)]]
+            inserted_new_records_params[0],
+            BTreeSet::from([expected_tx])
+        );
+        let pending_payable_notify_later_params =
+            pending_payable_notify_later_params_arc.lock().unwrap();
+        assert_eq!(
+            *pending_payable_notify_later_params,
+            vec![(ScanForPendingPayables::default(), pending_payable_interval)]
+        );
+        // The accountant is unbound here. We don't use the bind message. It means we can prove
+        // none of those other scan requests could have been sent (especially ScanForNewPayables,
+        // ScanForRetryPayables)
```

Comment:
Nonsense. Delete it.

You can add
```
        subject.scan_schedulers.payable.new_payable = Box::new(
            NotifyHandleMock::default().panic_on_call()
        );
        subject.scan_schedulers.payable.new_payable_later = Box::new(
            NotifyLaterHandleMock::default().panic_on_call()
        );
        subject.scan_schedulers.payable.retry_payable = Box::new(
            NotifyHandleMock::default().panic_on_call()
        );
        subject.scan_schedulers.payable.receivable = Box::new(
            NotifyHandleMock::default().panic_on_call()
        );
```

Warning: it is just pseudo code and may not compile like this...search for very similar namings if so